arches_controlled_lists/src/arches_controlled_lists/components/editor/ItemImages.vue

```diff
@@ -66,36 +66,40 @@ const showError = (event?: FileUploadErrorEvent | FileUploadUploadEvent) => {
 
 <template>
     <div class="images-container">
-        <h4>{{ $gettext("Images") }}</h4>
-        <FileUpload
-            accept="image/*"
-            :url="arches.urls.controlled_list_item_image_add"
-            :auto="true"
-            :max-file-size="5e6"
-            :file-limit="10"
-            :preview-width="250"
-            :with-credentials="true"
-            :show-cancel-button="false"
-            :show-upload-button="false"
-            :choose-button-props="{
-                severity: shouldUseContrast() ? CONTRAST : PRIMARY,
-            }"
-            choose-icon="fa fa-plus-circle"
-            :choose-label="$gettext('Upload an image')"
-            name="item_image"
-            :pt="{
-                content: ({ props, state }: FileUploadInternals) => {
-                    const done = [0, 100].includes(state.progress);
-                    return {
-                        style: { display: done ? 'none' : '' },
-                    };
-                },
-                pcChooseButton: { root: { style: { fontSize: 'smaller' } } },
-            }"
-            @before-send="addHeader($event)"
-            @upload="upload($event)"
-            @error="showError($event)"
-        />
+        <div class="images-container-title">
+            <h4>{{ $gettext("Images") }}</h4>
+            <FileUpload
+                class="add-image"
+                accept="image/*"
+                :url="arches.urls.controlled_list_item_image_add"
+                :auto="true"
+                :max-file-size="5e6"
+                :file-limit="10"
+                :preview-width="250"
+                :with-credentials="true"
+                :show-cancel-button="false"
+                :show-upload-button="false"
+                :choose-button-props="{
+                    severity: shouldUseContrast() ? CONTRAST : SECONDARY,
```

Comment:
Same here with the imports, for everything to build happily you'll need to remove the `PRIMARY` import and replace it with `SECONDARY`